taichi/backends/vulkan/vulkan_device.cpp

```diff
@@ -2039,6 +2039,10 @@ void VulkanSurface::destroy_swap_chain() {
   vkDestroySwapchainKHR(device_->vk_device(), swapchain_, nullptr);
 }
 
+int VulkanSurface::get_image_count() {
+  return swapchain_images_.size();
```

Comment:
In headless mode, `swapchain_images_` is empty, and there's only a single image (`screenshot_image_`)

Reply:
@AmesingFlank , should i mark this as resolved and get the PR in?